recipe/meta.yaml

```
@@ -48,33 +53,37 @@ requirements:
     - make                          # [unix]
     - m2-make                       # [win]
     - {{ compiler('c') }}           # [unix]
-    - {{ stdlib('c') }}             # [unix]
     - {{ compiler('m2w64_c') }}     # [win]
-    - {{ stdlib('m2w64_c') }}       # [win]
     - autoconf                      # [unix]
     - automake                      # [unix]
     - gettext                       # [unix]
     - libtool                       # [unix]
     - sed                           # [unix]
   host:
-    - xorg-xorgproto
     - libxcb
     - pthread-stubs
+    - xorg-xextproto
+    - xorg-kbproto
+    - xorg-inputproto
     - xorg-util-macros
+    - xorg-xorgproto
     - xorg-xtrans
+  run:
+    - xorg-xorgproto
+    - xorg-xproto
```

```suggestion
    - xorg-xorgproto
```
I'm not sure how we made it this far before this was caught but I think we should not be using `xorg-xproto` at all. It looks like `xorg-xorgproto` is the current modern package and it contains a superset of what can be found in `xorg-xproto`

---

In fact, I am seeing that `xorg-inputproto`, `xorg-xextproto` and `xorg-xproto` are also a part of `xorg-xorgproto` and should probably not exist as standalone packages.

---

Good catch. Thanks.

---

We probably need to go back to `xorg-xorgproto` and do something like this
```
run_constrained:
        - xorg-applewmproto <0.0.0a
        - xorg-bigreqsproto <0.0.0a
        - xorg-compositeproto <0.0.0a
        - xorg-damageproto <0.0.0a
        - xorg-dmxproto <0.0.0a
        - xorg-dpmsproto <0.0.0a
        - xorg-dri2proto <0.0.0a
        - xorg-dri3proto <0.0.0a
        - xorg-fixesproto <0.0.0a
        - xorg-fontsproto <0.0.0a
        - xorg-glproto <0.0.0a
        - xorg-inputproto <0.0.0a
        - xorg-kbproto <0.0.0a
        - xorg-presentproto <0.0.0a
        - xorg-randrproto <0.0.0a
        - xorg-recordproto <0.0.0a
        - xorg-renderproto <0.0.0a
        - xorg-resourceproto <0.0.0a
        - xorg-scrnsaverproto <0.0.0a
        - xorg-videoproto <0.0.0a
        - xorg-xcmiscproto <0.0.0a
        - xorg-xextproto <0.0.0a
        - xorg-xf86bigfontproto <0.0.0a
        - xorg-xf86dgaproto <0.0.0a
        - xorg-xf86driproto <0.0.0a
        - xorg-xf86vidmodeproto <0.0.0a
        - xorg-xineramaproto <0.0.0a
        - xorg-xproto <0.0.0a
        - xorg-xwaylandproto <0.0.0a
```

---

Maybe one of these as well
```
run_exports:
        - {{ pin_subpackage('xorg-xorgproto') }}
        - xorg-applewmproto >={{ version }}
        - xorg-bigreqsproto >={{ version }}
        - xorg-compositeproto >={{ version }}
        - xorg-damageproto >={{ version }}
        - xorg-dmxproto >={{ version }}
        - xorg-dpmsproto >={{ version }}
        - xorg-dri2proto >={{ version }}
        - xorg-dri3proto >={{ version }}
        - xorg-fixesproto >={{ version }}
        - xorg-fontsproto >={{ version }}
        - xorg-glproto >={{ version }}
        - xorg-inputproto >={{ version }}
        - xorg-kbproto >={{ version }}
        - xorg-presentproto >={{ version }}
        - xorg-randrproto >={{ version }}
        - xorg-recordproto >={{ version }}
        - xorg-renderproto >={{ version }}
        - xorg-resourceproto >={{ version }}
        - xorg-scrnsaverproto >={{ version }}
        - xorg-videoproto >={{ version }}
        - xorg-xcmiscproto >={{ version }}
        - xorg-xextproto >={{ version }}
        - xorg-xf86bigfontproto >={{ version }}
        - xorg-xf86dgaproto >={{ version }}
        - xorg-xf86driproto >={{ version }}
        - xorg-xf86vidmodeproto >={{ version }}
        - xorg-xineramaproto >={{ version }}
        - xorg-xproto >={{ version }}
        - xorg-xwaylandproto >={{ version }}
```

---

Here we go https://github.com/AnacondaRecipes/xorg-xorgproto-feedstock/pull/2